src/main/java/frc/robot/subsystems/ExampleSubsystem.java

```diff
@@ -3,23 +3,12 @@
 // the WPILib BSD license file in the root directory of this project.
 
 package frc.robot.subsystems;
-
-import edu.wpi.first.wpilibj.motorcontrol.PWMSparkMax;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 public class ExampleSubsystem extends SubsystemBase implements AutoCloseable {
 
-  // FIXME using PWMSparkMax because CANSparkMax doesn't have an equivalent simulation class
-  // May limit how much we can do in terms of JUnit tests
-  private PWMSparkMax motor;
-
   /** Creates a new ExampleSubsystem. */
   public ExampleSubsystem() {
-    motor = new PWMSparkMax(0);
```

Comment:
Was this conflicting with the PWM port you wanted to use?
Its probably as good a time as any to remove the ExampleSubsystem stuff anyway #31